payment-options-extension/pom-as-dependency.xml

```diff
@@ -0,0 +1,102 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Hi @Goutama , perhaps we remove pom-as-dependency.xml file completely as not much of reason to use extension pom as a dependency I guess. Such pom configuration can be used in projects to transitively include all options extension dependencies, which projects often prefer to do that themselves explicitly. What do you think?

Reply:
I saw comment on [pom.xml](https://github.com/Backbase/payments-examples/pull/2/files#diff-f4be196db5dcaf63b0444fe6ffd573e4877b5064d4415991c758fee52fce6c03R5), so added it. I will adjust comment and remove the pom-as-dependency.xml file.